openfe/protocols/restraint_utils/settings.py

```diff
@@ -0,0 +1,177 @@
+
+    The restraint is defined in the following manner:
+
+      H2                         G2
+       -                        -
+        -                      -
+         H1 - - H0 -- G0 - - G1
+
+    Where HX represents the X index of ``host_atoms``
+    and GX the X indexx of ``guest_atoms``.
+
+    By default, the Boresch-like restraint will be
+    obtained using a modified version of the
+    search algorithm implemented by Baumann et al. [1].
+
+    If ``guest_atoms`` and ``host_atoms`` are defined,
+    these indices will be used instead.
+
+    References
+    ----------
+    [1] Baumann, Hannah M., et al. "Broadening the scope of binding free
+        energy calculations using a Separated Topologies approach." (2023).
+    """
+    K_r: FloatQuantity['kilojoule_per_mole / nm ** 2'] = 8368.0 * unit.kilojoule_per_mole / unit.nm ** 2
+    """
+    The bond spring constant between H0 and G0.
+    """
+    K_thetaA: FloatQuantity['kilojoule_per_mole / radians ** 2'] = 83.68 * unit.kilojoule_per_mole / unit.radians ** 2
+    """
+    The spring constant for the angle formed by H1-H0-G0.
+    """
+    K_thetaB: FloatQuantity['kilojoule_per_mole / radians ** 2'] = 83.68 * unit.kilojoule_per_mole / unit.radians ** 2
+    """
+    The spring constant for the angle formed by H0-G0-G1.
+    """
+    K_phiA: FloatQuantity['kilojoule_per_mole / radians ** 2'] = 83.68 * unit.kilojoule_per_mole / unit.radians ** 2
+    """
+    The equilibrium force constant for the dihedral formed by
+    H2-H1-H0-G0.
+    """
+    K_phiB: FloatQuantity['kilojoule_per_mole / radians ** 2'] = 83.68 * unit.kilojoule_per_mole / unit.radians ** 2
+    """
+    The equilibrium force constant for the dihedral formed by
+    H1-H0-G0-G1.
+    """
+    K_phiC: FloatQuantity['kilojoule_per_mole / radians ** 2'] = 83.68 * unit.kilojoule_per_mole / unit.radians ** 2
+    """
+    The equilibrium force constant for the dihedral formed by
+    H0-G0-G1-G2.
+    """
+    host_selection: str = "backbone"
+    """
+    Boresch-like restraint search parameter.
+    An MDAnalysis selection string to sub-select the host atoms which will be involved in the restraint.
+    """
+    dssp_filter: bool = True
+    """
+    Boresch-like restraint search parameter.
+    Whether or not to try to do a DSSP filter on the host atoms.
+    """
+    rmsf_cutoff: Optional[FloatQuantity['nanometer']] = 0.1 * unit.nanometer
+    """
+    Boresch-like restraint search parameter.
+    The cutoff value for filtering atoms by their root mean square fluctuation. Atoms with values above this cutoff will be disregarded.
+    """
+    host_min_distance: Optional[FloatQuantity['nanometer']] = 1 * unit.nanometer
+    """
+    Boresch-like restraint search parameter.
+    The minimum distance between any host atom and the guest G0 atom. Must be in units compatible with nanometer.
+    """
+    host_max_distance: Optional[FloatQuantity['nanometer']] = 3 * unit.nanometer
```

Comment:
```suggestion
    host_max_distance: Optional[FloatQuantity['nanometer']] = 3.0 * unit.nanometer
```
For Protocol json roundtrip